docs/configuration/monitoring.md

```diff
@@ -1,43 +1,140 @@
-can be accessed using the Prometheus + Grafana ecosystem.
+Currently two monitoring solutions are supported:
 
-Initially Crow will still be bundled in Codacy helm chart, until
-we have all functionality available on Grafana.
+* [**Crow**](##Setting-up-monitoring-using-Crow) - A simple, lightweight, and builtin monitoring solution.
+* [**Prometheus+Grafana+Loki**](##Setting-up-monitoring-using-Grafana,-Prometheus,-and-Loki) - A more comprehensive third-party monitoring solution, for more advanced usage.
 
-## Setting up monitoring using Grafana & Prometheus
+## Setting up monitoring using Crow
+
+Crow is a visualization tool that displays information about the projects and jobs
+that are pending for analysis, as well as the ones running in the Codacy platform.
+
+The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
+It can be accessed through the `/monitoring` path of the url pointing to your Codacy
+installation, e.g. `http://codacy.company.org/monitoring`. You must set the
+`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
+projects can be properly established inside `crow`.
+
+For example:
+
+```yaml
+global:
+  codacy:
+    crow:
+      url: "http://codacy.example.com/monitoring"
+```
+
+Please see the [README.md](https://github.com/codacy/chart/blob/master/README.md) for more information about these values.
+
+### Configuring your credentials
+
+You **should** provide a password for the `crow` installation either through the `values.yaml` file or through a `--set` parameter during the `helm` installation process. This parameter can be configured as follows:
+
+* Through a `--set` parameter:
+
+```yaml
+helm upgrade (...) --set crow.config.passwordAuth.password=<--- crow password --->
+```
+
+* Through the `values.yaml` file:
+
+```yaml
+crow:
+  config:
+    passwordAuth:
+      password: <--- crow password --->
+```
+
+**If you have not configured your `crow` credentials as described above
+the following default credentials will be used:**
+
+```yaml
+username: codacy
+password: C0dacy123
+```
+
+## Setting up monitoring using Grafana, Prometheus, and Loki
+
+[Prometheus](https://prometheus.io) is an open-source systems monitoring and alerting
+toolkit. Logs can be collected using [Loki](https://grafana.com/oss/loki/), which is a
+horizontally-scalable, highly-available, multi-tenant log aggregation system
+It's data can be visualized with [Grafana](https://grafana.com), a widely used
+open source analytics and monitoring solution.
+
+The following guide covers the basic installation of the components. This monitoring
+stack is considerably more resource demanding than Crow, and is recommended only for
+more advanced usage. Furthermore, its installation, configuration, and management
+requires a deeper knowledge of Kubernetes as each component must be carefully tweaked
+to match your specific use case, using as starting point the values files herein provided.
```

Comment:
```suggestion
The following guide covers the basic installation of the components in this monitoring stack.
This solution is considerably more resource demanding than Crow, and is recommended only for
more advanced usage. Furthermore, its installation, configuration, and management
requires a deeper knowledge of Kubernetes as each component must be carefully tweaked
to match your specific use case, using as starting point the .yaml values files provided by us.
```

Reply:
Herein is more informative because, while it implies that the files are provided by us, it also states that they are in this repo and not elsewhere. The rest is somewhat equivalent, but I don't mind changing it.